### PR TITLE
Remove now aliases nixUnstable

### PR DESCRIPTION
**Pull request by SuperSandro2000**
@Mic92 or do you have a better idea?

### CODE REVIEW

File: default.nix [hunk 1]
```diff
@@ -15,7 +15,7 @@ python3.pkgs.buildPythonApplication rec {
 
     # needed for interactive unittests
     python3.pkgs.pytest
-    nix_2_4
+    nixVersions.nix_2_4
```
Comment (Mic92) [inline default.nix:18]: Does this work in 21.11 as well?

Comment (SuperSandro2000) [reply default.nix:18]: I don't think so but I also don't know how to make it work in both versions.

Comment (jtojnar) [reply default.nix:18]: You could use `pkgs.nixVersions.nix_2_4 or nix_2_4`.

Comment (SuperSandro2000) [reply default.nix:18]: That does not work because nixVersions is also undefined.

Comment (jtojnar) [reply default.nix]: the `or` syntax should work even for nested attribute sets.

Comment (SuperSandro2000) [reply default.nix]: Now it works. I tried that before and I am sure that it didn't work.

**Pull request merged**